py/torch_tensorrt/dynamo/conversion/_TRTInterpreter.py

```diff
@@ -201,12 +203,20 @@ def run(
                     self.compilation_settings.optimization_level
                 )
 
-        builder_config.engine_capability = self.compilation_settings.engine_capability
+        builder_config.engine_capability = (
+            self.compilation_settings.engine_capability.to(trt.EngineCapability)
+        )
         builder_config.avg_timing_iterations = (
             self.compilation_settings.num_avg_timing_iters
         )
 
         if self.compilation_settings.device.device_type == trt.DeviceType.DLA:
+            device_info = torch.cuda.get_device_properties(
+                self.compilation_settings.device.gpu_id
+            )
+            assert (device_info.major == 8 and device_info.minor == 7) or (
+                device_info.major == 7 and device_info.minor == 2
+            )
```

Comment:
Just to verify - should these be `>=` ranges, or is it a fixed set?

Reply:
Its a fixed set these are 2 specific SM versions that key to AGX Orin and AGX Xavier